build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -71,8 +71,8 @@ spec:
       containers:
       {{- end }}
         - name: install-cni
-          image: {{ include "antreaImage" . | quote }}
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          image: {{ include "antreaAgentImage" . | quote }}
+          imagePullPolicy: {{ .Values.agentImage.pullPolicy }}
```

Comment:
the PullPolicy needs a template for consistency: if `.Values.image` is non-empty, then `.Values.image.pullPolicy` should be used, otherwise `.Values.agentImage.pullPolicy` should be used

same comment for instances below